site/content/docs/main/restore-hooks.md

```diff
@@ -0,0 +1,240 @@
+  "volumeMounts": [
+    {
+      "mountPath": "/restores/pvc1-vm",
+      "name": "pvc1-vm"
+    }
+  ]
+  ...
+}
+```
+
+and
+
+```json
+{
+  "command": [
+    "/bin/ash",
+    "-c",
+    "echo -n \"DEADFEED\" >> /restores/pvc2-vm/deadfeed"
+  ],
+  "image": "alpine:latest",
+  "imagePullPolicy": "Always",
+  "name": "restore-hook-init2",
+  "resources": {},
+  "terminationMessagePath": "/dev/termination-log",
+  "terminationMessagePolicy": "File",
+  "volumeMounts": [
+    {
+      "mountPath": "/restores/pvc2-vm",
+      "name": "pvc2-vm"
+    }
+  ]
+  ...
+}
+```
+
+## Exec Restore Hooks
+
+Use an Exec Restore hook to execute commands in a restored pod's containers after they start.
+
+There are two ways to specify `Exec` restore hooks:
+1. Specifying exec restore hooks in annotations
+1. Specifying exec restore hooks in the restore spec
+
+### Specifying Exec Restore Hooks As Pod Annotations
+
+Below are the annotations that can be added to a pod to specify exec restore hooks:
+* `post.hook.restore.velero.io/container`
+    * The container name where the hook will be executed. Defaults to the first container.
+* `post.hook.restore.velero.io/command`
+    * The command that will be executed in the container. Required.
+* `post.hook.restore.velero.io/on-error`
+    * How to handle execution failures. Valid values are `Fail` and `Continue`. Defaults to `Continue`. With `Continue` mode, execution failures are logged only. With `Fail` mode, no more restore hooks will be executed in any container in any pod and the status of the Restore will be `PartiallyFailed`.
+* `post.hook.restore.velero.io/exec-timeout`
+    * How long to wait once execution begins.
+* `post.hook.restore.velero.io/wait-timeout`
+    * How long to wait for a container to become ready. This should be long enough for the container to start plus any preceding hooks in the same container to complete. The wait timeout begins when the container is restored and may require time for the image to pull and volumes to mount. If not set the restore will wait indefinitely.
+
+#### Example
+
+Use the below commands to add annotations to the pods before taking a backup.
+
+```bash
+$ kubectl annotate pod -n <POD_NAMESPACE> <POD_NAME> \
+    post.hook.restore.velero.io/container=postgres \
+    post.hook.restore.velero.io/command='["/bin/bash", "-c", "psql < /backup/backup.sql"]' \
+    post.hook.restore.velero.io/wait-timeout=5m \
+    post.hook.restore.velero.io/exec-timeout=45s \
+    post.hook.restore.velero.io/on-error=Continue
+```
+
+### Specifying Exec Restore Hooks in Restore Spec
+
+Exec restore hooks can also be specified using the `RestoreSpec`.
+Please refer to the documentation on the [Restore API Type][1] for how to specify hooks in the Restore spec.
+
+#### Example
+
+Below is an example of specifying restore hooks in  a `RestoreSpec`.
+When using the restore spec it is possible to specify multiple hooks for a single pod, as this example demonstrates.
```

Comment:
When using multiple hooks for a single pod, do they run in order like the initcontainers? It might be helpful to give a few more details about the execution order.